glean-core/rlb/src/net/mod.rs

```diff
@@ -0,0 +1,112 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! A module responsible for the Glean upload logic.
+//!
+//! This doesn't perform the actual upload but rather handles
+//! retries, upload limitations and error tracking.
+
+use std::sync::{
+    atomic::{AtomicBool, Ordering},
+    Arc,
+};
+use std::thread;
+use std::time::Duration;
+
+use crate::with_glean;
+use glean_core::upload::PingUploadTask;
+pub use glean_core::upload::{PingRequest, UploadResult};
+
+pub use http_uploader::*;
+
+mod http_uploader;
+
+/// The number of milliseconds to make the uploader thread, when told to,
+/// by glean-core.
+const THROTTLE_BACKOFF_TIME_MS: u64 = 60_000;
```

Comment:
Use proper types and not variable suffixes to type things (instantiation functions of `Duration` are all const)
```suggestion
const THROTTLE_BACKOFF_TIME: Duration = Duration::from_secs(60);
```